File: sop-frontend/src/app/_models/order.ts
import { User } from './user';
import { Product } from './product';

export class Order {
    id: number;
    name: string;
    address: string;
    courierId: number;
    customerId: number;
    orderStatus: string;
    products: Array<Product>;
    total: number;
}

File: sop-backend/src/main/java/com/sop/sopbackend/dto/Order.java
package com.sop.sopbackend.dto;

import lombok.Data;

import java.math.BigDecimal;
import java.util.List;

@Data
public class Order {

    private Integer id;

    private Long customerId;

    private Long courierId;

    private List<Product> products;

    private String address;

    private BigDecimal total;

    private OrderStatusEnum orderStatus;
}
